doc/nrf/doc_styleguide.rst

```diff
@@ -67,6 +82,10 @@ See the following example for including a doxygen group:
       :project: nrf
       :members:
 
+
+.. note::
+    Including a group does not include all its subgroups. You must list the subgroups on the group page.
+
 The `Breathe documentation`_ contains information about what you can link to.
 
 To link directly to a Doxygen reference from RST, use the following breathe
```

Comment:
While you're updating this file, can you fix the example in the note a bit further down.
Instead of ``:cpp:enumerator:`ENUM_VALUE <DOXYGEN_GROUP>::ENUM_VALUE` ``, it should say ``:cpp:enumerator:`ENUM_VALUE <DOXYGEN_GROUP::ENUM_VALUE>` ``.